spec/requests/link_checker_api_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
```

Comment:
Local links manager doesn't seem to be as strict on the quotes as `publishing-api`...